network.bisq.Bisq.yml

```diff
@@ -0,0 +1,56 @@
+app-id: network.bisq.Bisq
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.openjdk17
+command: Bisq
+finish-args:
+    - "--share=network"
+    - "--share=ipc"
+    - "--socket=x11"
+    - "--socket=pulseaudio"
+    - "--device=dri"
+    - "--env=PATH=/usr/bin:/app/bin:/app/jdk/bin"
+    - "--env=JAVA_HOME=/app/jdk"
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+          - /usr/lib/sdk/openjdk17/installjdk.sh
+    - name: bisq
+      buildsystem: simple
+      build-commands:
+          - ar x Bisq-*.deb
+          - rm Bisq-*.deb
+          - tar xf data.tar.xz
+          - cp -r opt/bisq/lib /app/lib
+          - install -D opt/bisq/bin/Bisq /app/bin/Bisq
+      sources:
+          - type: file
+            sha256: 310fa6381914d327ae1c1da995a78cad01df7f9a9b15863ad257f55194af9457
+            url: "https://bisq.network/downloads/v1.7.5/Bisq-64bit-1.7.5.deb"
```

Comment:
is this platform independent?

Reply:
you should disable other architectures or build from source to support other than x86_64

Reply:
Thanks! fixed